lib/commands/build.rb

```diff
@@ -0,0 +1,9 @@
+require_relative './base'
+require_relative './compose'
+
+class Commands::Build < Commands::Base
+  def call
+    check_service_exists
+    system.call("make", "-f", "#{config_directory}/Makefile", service)
```

Comment:
Can we just do `make service` here?

Reply:
By passing the Makefile directly, you can run this from the app directory itself not just `govuk-docker`.

Reply:
Oh, I thought that's what @barrucadu fixed in https://github.com/alphagov/govuk-docker/pull/54?

Reply:
Ah, sorry - I see what you mean now. (It's not the service-specific one.)